docs/UserGuide.md

```diff
@@ -247,27 +261,45 @@ Format: `clearstats INDEX`
 Examples:
 * `list` followed by `clearstats 2` clears the statistics of the 2nd flashcard in the list.
 
-### Importing a set of flashcards : `import`
+### Sharing flashcards:
+
+#### Exporting a set of flashcards : `export FILE_NAME`
+
+<div markdown="block" class="alert alert-info">
+:bulb: The filename specified includes the file format extension e.g. file.json
```

Comment:
I think this information block should use the `:information_source:` ℹ️ emoji instead of a bulb.